packages/http-client/src/ceramic-http-client.ts

```diff
@@ -129,27 +132,32 @@ export default class CeramicClient implements CeramicApi {
 
   async createDocumentFromGenesis<T extends Doctype>(doctype: string, genesis: any, opts?: DocOpts): Promise<T> {
     const doc = await Document.createFromGenesis(this._apiUrl, doctype, genesis, this.context, opts, this._config)
-    const docIdStr = doc.id.toString()
-    if (!this._docmap[docIdStr]) {
-      this._docmap[docIdStr] = doc
-    } else if (!DoctypeUtils.statesEqual(doc.state, this._docmap[docIdStr].state)) {
-      this._docmap[docIdStr].state = doc.state
-      this._docmap[docIdStr].emit('change')
+
+    let docFromCache = this._docCache.get(doc.id) as Document
+    if (docFromCache == null) {
+      this._docCache.set(doc)
+      docFromCache = doc
+    } else if (!DoctypeUtils.statesEqual(doc.state, docFromCache.state)) {
+      docFromCache.state = doc.state
```

Comment:
does the call to `_docCache.get()` above reset the doc's location in the LRU cache order?  If not, we need to do something to do so explicitly here.

Reply:
Yes, it registers the recent use.